packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -193,6 +229,20 @@ export default {
       versionText: '',
       VIEW_COMPONENT,
       VIEW_FLOW,
+      filters: {
+        limitRootEvents: {
+          on: true,
+          default: true,
```

Comment:
The default of this filter should depend on whether or not the event array contains any HTTP server requests.

If there **aren't** any HTTP server requests, it should be off by default so that we don't display an empty Trace.
If there **are** HTTP server requests, it should be on by default to reduce noise.

Reply:
Done https://github.com/applandinc/appmap-js/pull/217/commits/4c26a606138eb215a3cd4b7d3ff78bf88c329d54